src/Microsoft.PackageManagement.NuGetProvider/project.json

```diff
@@ -3,7 +3,8 @@
 	"version": "1.0.0-*",
 
 	"buildOptions": {
-		"warningsAsErrors": true
+		"warningsAsErrors": true,
+                "debugType": "portable"
```

Comment:
the `debugType` setting should go in the `buildOptions` section within `netstandard1.6`, preferably right after `define`.